### PR TITLE
List files alphabetically in README.md

### PR DESCRIPTION
**Pull request by jeanettl**
Change-Id: Ib4647dafa9ae13b46ec010f0d331c329a88c2730

### CODE REVIEW

File: licensekey/README.md [hunk 1]
```diff
@@ -98,8 +98,6 @@ http://<axis_device_ip>/#settings/apps
 *Goto your device web page above > Click on the tab **Apps** in the device GUI > Add **(+)** sign and browse to
 the newly built **licensekey_handler_1_0_0_armv7hf.eap** > Click **Install** > Run the application by enabling the **Start** switch*
 
-Application will run with default video compression format h264.
```
Comment (pataxis) [inline licensekey/README.md:101]: Was this removal intentional?

File: vdo-larod/README.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ vdo-larod
 ├── yuv
 │   ├── 0001-Create-a-shared-library.patch
 │   ├── build.sh
-│   ├── Dockerfile
+└── └── Dockerfile
```
Comment (pataxis) [inline vdo-larod/README.md:67]: I think the inner └── could be removed, leaving
`
└── yuv
     ├── 0001-Create-a-shared-library.patch
     ├── build.sh
     └── Dockerfile
`

File: vdo-larod/README.md [hunk 3]
```diff
@@ -149,7 +149,7 @@ vdo-larod
 │   ├── vdo_larod*
 │   ├── vdo_larod.c
 │   ├── vdo_larod_cpu_1_0_0_armv7hf.eap / vdo_larod_edgetpu_1_0_0_armv7hf.eap
-│   ├── vdo_larod_1_0_0_LICENSE.txt
+└── └── vdo_larod_1_0_0_LICENSE.txt
```
Comment (pataxis) [inline vdo-larod/README.md:152]: I think the inner └── could be removed, see previous comment

File: vdostream/README.md [hunk 4]
```diff
@@ -120,26 +120,26 @@ head -50 info.log
...
-2020-06-22T11:00:03.872+02:00 axis-accc8efc9c84 [ INFO    ] vdoencodeclient[17690]: frame = 35740, type = P, size = 86
-2020-06-22T11:00:03.905+02:00 axis-accc8efc9c84 [ INFO    ] vdoencodeclient[17690]: frame = 35741, type = P, size = 79
-2020-06-22T11:00:03.938+02:00 axis-accc8efc9c84 [ INFO    ] vdoencodeclient[17690]: frame = 35742, type = P, size = 78
-2020-06-22T11:00:03.972+02:00 axis-accc8efc9c84 [ INFO    ] vdoencodeclient[17690]: frame = 35743, type = P, size = 77
-2020-06-22T11:00:04.005+02:00 axis-accc8efc9c84 [ INFO    ] vdoencodeclient[17690]: frame = 35744, type = P, size = 71
-2020-06-22T11:00:04.038+02:00 axis-accc8efc9c84 [ INFO    ] vdoencodeclient[17690]: frame = 35745, type = P, size = 82
-2020-06-22T11:00:04.072+02:00 axis-accc8efc9c84 [ INFO    ] vdoencodeclient[17690]: frame = 35746, type = P, size = 75
-2020-06-22T11:00:04.105+02:00 axis-accc8efc9c84 [ INFO    ] vdoencodeclient[17690]: frame = 35747, type = P, size = 77
-2020-06-22T11:00:04.138+02:00 axis-accc8efc9c84 [ INFO    ] vdoencodeclient[17690]: frame = 35748, type = P, size = 80
+11:00:03.509 [ INFO    ] vdoencodeclient[17690]: frame = 35729, type = P, size = 81
```
Comment (pataxis) [inline vdostream/README.md:123]: Nit: Not very necessary after log output has shrunk. But if logs still feel very wide, an additional fix could be to,
[ INFO    ] -> [INFO]
vdoencodeclient[17690] -> vdoencodeclient[690]

**Review by kimraxis — APPROVED** [review]

File: vdo-larod/README.md (hunk 2 above)
Comment (pataxis) [reply vdo-larod/README.md:67]: Hmm, Github mess up my comment , I hope it's understandable from the text instead...

File: vdo-larod/README.md (hunk 3 above)
Comment (pataxis) [reply vdo-larod/README.md:152]: Same here Github mess up my comment , I mean all the first characters | could be removed

File: licensekey/README.md (hunk 1 above)
Comment (jeanettl) [reply licensekey/README.md:101]: Yes this line is not applicable for this example.

File: vdo-larod/README.md (hunk 2 above)
Comment (jeanettl) [reply vdo-larod/README.md:67]: Done

File: vdo-larod/README.md (hunk 3 above)
Comment (jeanettl) [reply vdo-larod/README.md:152]: Done

File: vdostream/README.md (hunk 4 above)
Comment (jeanettl) [reply vdostream/README.md:123]: Done

**Review by mirzamah — APPROVED** [review]

**Review by pataxis — APPROVED** [review]
